modules/ts/src/ts.cpp

```diff
@@ -684,7 +688,11 @@ void parseCustomOptions(int argc, char **argv)
 
     test_ipp_check = parser.get<bool>("test_ipp_check");
     if (!test_ipp_check)
+#ifndef WINRT
         test_ipp_check = getenv("OPENCV_IPP_CHECK") != NULL;
```

Comment:
Is file system accessible on WINRT?
What is about command line parameter to specify "test_env.conf" and transparent replace `getenv` to read this file?
What is about to pass `--env MY_ENV_VAR=MY_ENV_VALUE` via command line?